bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="boschshc"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Bosch Bridge -->
+	<bridge-type id="shc">
+		<label>Smart Home Controller</label>
+		<description>The Bosch SHC Bridge representing the Bosch Smart Home Controller.</description>
+
+		<config-description-ref uri="thing-type:boschshc:bridge"/>
+	</bridge-type>
+
+	<thing-type id="in-wall-switch">
+		<label>In-wall Switch</label>
+		<description>Bosch In-wall switch for light control</description>
+
+		<channels>
+			<channel id="power-switch" typeId="system.power"/>
+			<channel id="power-consumption" typeId="power-consumption"/>
+			<channel id="energy-consumption" typeId="energy-consumption"/>
+		</channels>
+
+		<config-description-ref uri="thing-type:boschshc:device"/>
+
+	</thing-type>
+
+	<thing-type id="twinguard">
+		<label>TwinGuard</label>
+		<description>Bosch TwinGuard environmental sensor</description>
+
+		<channels>
+			<channel id="temperature" typeId="temperature"/>
+			<channel id="temperature-rating" typeId="temperature-rating"/>
+			<channel id="humidity" typeId="humidity"/>
+			<channel id="humidity-rating" typeId="humidity-rating"/>
+			<channel id="purity" typeId="purity"/>
+			<channel id="air-description" typeId="air-description"/>
+			<channel id="purity-rating" typeId="purity-rating"/>
+			<channel id="combined-rating" typeId="combined-rating"/>
+		</channels>
+
+		<config-description-ref uri="thing-type:boschshc:device"/>
+
+	</thing-type>
+
+	<thing-type id="window-contact">
+		<label>Window/Door Contact</label>
+		<description>Bosch Contact for windows and doors</description>
+
+		<channels>
+			<channel id="contact" typeId="contact"/>
+		</channels>
+
+		<config-description-ref uri="thing-type:boschshc:device"/>
+
+	</thing-type>
+
+	<thing-type id="motion-detector">
+		<label>Motion Detector</label>
+		<description>Bosch Motion Detector</description>
+
+		<channels>
+			<channel id="latest-motion" typeId="latest-motion"/>
+		</channels>
```

Comment:
How would you visualize the motion in a sitemap? Many motion sensors have a `Contact` Channel, which you can map to the motion icon easily.

Reply:
The motion detector reports the `DateTime` of the latest detected motion. Maybe what you think about is the WindowContactHandler? That one has a `Contact` channel (called `contact` :) ).

Reply:
I'm still wondering how you would control e.g. the motion icon without a rule. But that's only a functional issue, which is up to you.

Reply:
Okay, I like to keep it like this for now. I like to forward the data from the Bosch controller as precise as it is for now. We might add some additional channels with custom logic if needed.